packages/client/src/features/accessibility/view-key-tools/grid-zoom-key-tool.ts

```diff
@@ -75,18 +75,15 @@ export class ZoomKeyTool implements Tool {
         if (isViewport(this.editorContextService.modelRoot)) {
             let viewportAction: Action | undefined = undefined;
 
-            if (KeyboardGridCellSelectedAction.is(action) && action.options.originId === ZoomKeyTool.ID) {
+            if (KeyboardGridCellSelectedAction.is(action) && action.options.originId === GridZoomTool.ID) {
                 viewportAction = this.zoomKeyListener.setNewZoomFactor(
                     this.editorContextService.modelRoot,
-                    ZoomKeyListener.defaultZoomInFactor,
+                    ZoomFactor.Default.IN,
                     getAbsolutePositionByPoint(this.editorContextService.modelRoot, action.options.centerCellPosition)
                 );
-            } else if (KeyboardGridKeyboardEventAction.is(action) && action.options.originId === ZoomKeyTool.ID) {
+            } else if (KeyboardGridKeyboardEventAction.is(action) && action.options.originId === GridZoomTool.ID) {
                 if (matchesKeystroke(action.options.event, 'Minus')) {
-                    viewportAction = this.zoomKeyListener.setNewZoomFactor(
-                        this.editorContextService.modelRoot,
-                        ZoomKeyListener.defaultZoomOutFactor
-                    );
+                    viewportAction = this.zoomKeyListener.setNewZoomFactor(this.editorContextService.modelRoot, ZoomFactor.Default.OUT);
```

Comment:
Maybe we should provide the ZoomFactor. in/out properties via a injectable options?
This would make it easier for adopters to customize